commands/cmdutils/cmdutils.go

```diff
@@ -174,6 +174,46 @@ func LabelsPrompt(response *[]string, apiClient *gitlab.Client, repoRemote *glre
 	return nil
 }
 
+func MilestonesPrompt(response *int, apiClient *gitlab.Client, repoRemote *glrepo.Remote) (err error) {
+	var addMilestones bool
+	err = prompt.Confirm(&addMilestones, "Do you wish to attach a milestone?", true)
+	if err != nil {
+		return err
+	}
+	if addMilestones {
+		var milestoneOptions []string
+		milestoneMap := map[string]*gitlab.Milestone{}
+
+		lOpts := &gitlab.ListMilestonesOptions{
+			State: gitlab.String("active"),
+		}
+		lOpts.PerPage = 100
+		milestones, err := api.ListMilestones(apiClient, repoRemote.FullName(), lOpts)
+		if err != nil {
+			return err
+		}
+		if len(milestones) == 0 {
+			return fmt.Errorf("There are no active milestones in this project")
```

Comment:
I'm thinking instead of returning an error which will cause the user to lose info entered, we just print the message to stderr and return nil; this will help skip this step and continue with the submission so there will be no data loss (since there's no way to recover and continue from previous point as there is in gh).

That's my suggestion. What do you think?

If implemented, you may have to change the signature of the function to include the `*utils.IOStream` in the arguments
```go
func MilestonesPrompt(response *int, apiClient *gitlab.Client, repoRemote *glrepo.Remote, ios *utils.IOStream) (err error) {
```
Then the error printed with:
```go
fmt.Fprintln(ios.Stderr, "There are no active milestones in this project")
return nil
```

Reply:
Done